lib/Docs/CodeBlockWithLineNumbersRenderer.php

```diff
@@ -107,7 +107,7 @@ private function generateLineNumbersTableTemplate(array $lines, string $codeElem
         $lineTableRows = [];
 
         foreach ($lines as $key => $line) {
-            $lineNumber = $key + 1;
+            $lineNumber = (int) $key + 1;
```

Comment:
This should be a documentation on `$lines`: if you document it as `@var array <int, Foo>` it will be OK. A cast is not necessary